src/environment.ts

```diff
@@ -302,6 +304,8 @@ export class Environment {
   private evaluateFeature<K extends keyof Features>(feature: K): Features[K] {
     if (feature === 'DEBUG') {
       return false;
+    } else if (feature === 'IS_NODE') {
```

Comment:
Given this stanza it seems like it should be IS_BROWSER not IS_NODE.  You're checking if the window object exists, not whether we authoritatively know that we're in node.